src/js/fs.js

```diff
@@ -20,6 +20,10 @@ var util = require('util');
 var fsBuiltin = native;
 
 fs.exists = function(path, callback) {
+  if (!(util.isString(path)) && !(util.isBuffer(path))) {
+    throw new TypeError("Path should be a string or a buffer");
+    return;
```

Comment:
`return` should not be after the `throw` statement :(

Reply:
@yorkie You're right! @DanielBallaSZTE Please fix it.